src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,144 @@
+package duke;
```

Comment:
Overall, I like how you use appropriate language and avoid slangs or names that are not meaningful within contexts/times.